source/rules/roles.txt

```diff
@@ -240,21 +262,56 @@ data source's default roles.
               ]
             }
 
+With Device Sync
+~~~~~~~~~~~~~~~~
+
+When using Device Sync, App Services assigns a role at the beginning of each
+Sync session. The role determines which permissions apply for the duration of a
+Flexible Sync session.
```

Comment:
Specifying Flexible Sync in the second sentence, but not in the first feels odd.
```suggestion
When using Device Sync, App Services assigns a role at the beginning of each
Flexible Sync session. The role determines which permissions apply for the duration of a
session.
```